packages/fxa-profile-server/lib/routes/display_name/post-from-auth.js

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ const Joi = require('@hapi/joi');
+ const db = require('../../db');
+ const notifyProfileUpdated = require('../../updates-queue');
+
+ const EMPTY = Object.create(null);
+
+ // We're pretty liberal with what's allowed in a display-name,
```

Comment:
We should file a follow up to clean up the duplication and probably find a better name for this file.